src/plugins/workspace/public/components/workspace_list/index.test.tsx

```diff
@@ -24,11 +24,16 @@ jest.mock('../delete_workspace_modal', () => ({
   ),
 }));
 
+const defaultProps = ({
+  workspaceConfigurableApps$: of(undefined),
+} as unknown) as WorkspaceListProps;
```

Comment:
```suggestion
const defaultProps: WorkspaceListProps = {
  workspaceConfigurableApps$: of(undefined),
};
```
Can we do this? The cast `as WorksapaceListProps` is not necessary and safe here.

Reply:
The suggested change will also throw type error. I'm open to if we use type assertion in test case.

Reply:
1. It seems unnecessary, we can just mock an object that matches the WorkspaceListProps.
2. WorkspaceListProps may have more field and if we use type assertion, the type error will be swallowed.

Reply:
So do you mean if there is type error in test file we can ignore? Seems your suggested change will throw type error.

Reply:
No, I mean we should not use type assertion here. Could you please try the latest version of the suggestions? It should work without type error as long as you import type PublicAppInfo.

Reply:
Sure, that's fine.